dutree/dutree.py

```diff
@@ -304,10 +305,38 @@ class DuScan:
     "Disk Usage Tree scanner"
 
     def __init__(self, pathname):
+        self._dev_allow = []
+        self._dev_deny = []
+        self._dev = lstat(pathname).st_dev
         self._path = self._normpath(pathname)
         self._tree = None
         self._check_path()
 
+    def skip_other_filesystems(self):
+        "add self._dev to self._dev_allow"
+        self._dev_allow = [self._dev]
+
+    def skip_proc_sys_filesystems(self):
+        "add proc and sys devices to self._dev_deny"
+
+        try:
+            proc_device = lstat('/proc').st_dev
+        except Exception:
+            warnings.warn('/proc not found, so device is not skipped',
+                          OsWarning)
+        finally:
+            if not proc_device == self._dev:
```

Comment:
Does lint not remark anything about `(not A == B)` -> `(A != B)` ?

I could see an argument for allowing this, but it is a quite uncommon pattern, so.. no.